lib/grpc-api-assistant/grpc_steps.rb

```diff
@@ -122,6 +122,26 @@
   expect(value).to be_within(ms.to_i).of(Time.now.to_i * 1000)
 end
 
+Then /^the '(.+)' field in the response object has a timestamp at least (\d+) milliseconds old$/ do |field_path, ms|
+  expect(@grpc_response).not_to be nil
+  expect(@grpc_response).not_to be_a(GRPC::BadStatus)
+  value = GrpcHelpers::fetch_from_grpc_with_shorthand(field_path, @grpc_response)
+  expect(value).not_to be_within(ms.to_i).of(Time.now.to_i * 1000)
+end
+
+Then(/^the '(.+)' field in the response object has a timestamp less than (\d+) seconds old$/) do |field_path, seconds|
+  expect(@grpc_response).not_to be nil
+  expect(@grpc_response).not_to be_a(GRPC::BadStatus)
+  value = GrpcHelpers.fetch_from_grpc_with_shorthand(field_path, @grpc_response)
+  expect(value).to be_within(seconds.to_i).of(Time.now.to_i)
+end
+
+Then(/^the '(.+)' field in the response object has a timestamp at least (\d+) seconds old$/) do |field_path, seconds|
+  expect(@grpc_response).not_to be nil
+  expect(@grpc_response).not_to be_a(GRPC::BadStatus)
+  value = GrpcHelpers.fetch_from_grpc_with_shorthand(field_path, @grpc_response)
+  expect(value).not_to be_within(seconds.to_i).of(Time.now.to_i)
```

Comment:
Instead of copying the code used between this step and the one on line 125, it would be better to treat one of them as a convenience / shorthand.  For example, if the ms version is the "main" it might look like this:

```ruby
step "the '#{field_path}' in the response object has a timestamp at least #{seconds * 1000} milliseconds old"
```